docs/lsm/allocation/allocation_sources/allocation_v3/complex_example/v3_plugin.py

```diff
@@ -0,0 +1,25 @@
+@allocation_helpers.allocator()
+def get_free_value_in_range(
```

Comment:
The lsm module already has an equivalent allocator, it would be good to illustrate this one instead.